client/layout/masterbar/masterbar-cart/masterbar-cart-button.tsx

```diff
@@ -64,9 +64,10 @@ export function MasterbarCartButton( {
 				icon="cart"
 				tooltip={ tooltip }
 				onClick={ onClick }
+				isActive={ isActive }
 				ref={ cartButtonRef }
 			>
-				<MasterbarCartCount productsInCart={ responseCart.products.length } />
+				{ responseCart.products.length > 0 && <span className={ 'masterbar-cart__bubble' } /> }
```

Comment:
No need to add extra expression evaluation when we know it's going to be a string for sure.

```suggestion
				{ responseCart.products.length > 0 && <span className="masterbar-cart__bubble" /> }
```